src/styles/blocks/card.scss

```diff
@@ -0,0 +1,62 @@
+.card {
+  box-sizing: border-box;
+  position: relative;
+  text-align: justify;
+  width: 200px;
+  height: 407px;
+  border: 1px solid $grey-star-color;
+  border-radius: 5px;
+  transition: all 0.3s linear;
+}
+
+.card__product {
+  display: block;
+  padding-inline: $standart-inline-pad;
```

Comment:
The variable name '$standart-inline-pad' seems to have a typo. It should be '$standard-inline-pad' for correct spelling.